web/src/main/java/com/bazaarvoice/emodb/web/resources/blob/BlobStoreResource1.java

```diff
@@ -86,10 +92,22 @@
 
     private final BlobStore _blobStore;
     private final Set<String> _approvedContentTypes;
+    private final LoadingCache<String, Meter> _blobWritesWithTtlByTableName;
 
-    public BlobStoreResource1(BlobStore blobStore, Set<String> approvedContentTypes) {
+    public BlobStoreResource1(BlobStore blobStore, Set<String> approvedContentTypes, MetricRegistry metricRegistry) {
         _blobStore = blobStore;
         _approvedContentTypes = approvedContentTypes;
+        _blobWritesWithTtlByTableName = CacheBuilder.newBuilder().build(new CacheLoader<String, Meter>() {
+            @Override
+            public Meter load(String key) throws Exception {
+                String metricName = new TaggedName.TaggedNameBuilder()
+                        .metricName("bv.emodb.blob.BlobStore.blobWriteWithTTL")
+                        .addTag("table", key)
```

Comment:
we have 20200 tables in bazaar, but I don't expect that `TTL` feature is even used.
haven't found anything in `request.log`.
but will monitor memory consumption after deployment so we can rollback in case of issues.

Reply:
I agree, but what is the memory cost of having 20K separate metrics? If we are already pretty sure that it isn't used, it may be best to just keep it simple, and not add more overhead until are given a good reason to do so (i.e. we see a usage of it).

Reply:
Additionally, we would be able to avoid having to do risk analysis on upgrading the dependency (since we wouldnt have to upgrade it at all), and we could more quickly push this to production.

Reply:
i agree that on first pass we can have only total metric and add grouping that in future.
for dependency update, i would still keep it, as it's much easier to make dependency updates incrementally and that would be helpful for dropwizard update PR.